### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by megan07**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
 ## 3.76.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_certificate_manager_certificate` ([#9540](https://github.com/hashicorp/terraform-provider-google/pull/9540))
```
Comment (slevenick) [inline CHANGELOG.md:4]: These certificate_manager resources should be removed from the release if they haven't been already

Comment (ScottSuarez) [reply CHANGELOG.md:4]: https://github.com/hashicorp/terraform-provider-google/commits/release-3.76.0

Comment (ScottSuarez) [reply CHANGELOG.md:4]: looks good, they are out

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+
+FEATURES:
+* **New Resource:** `google_dialogflow_cx_flow` ([#9551](https://github.com/hashicorp/terraform-provider-google/pull/9551))
+* **New Resource:** `google_dialogflow_cx_intent` ([#9537](https://github.com/hashicorp/terraform-provider-google/pull/9537))
+* **New Resource:** `google_dialogflow_cx_version` ([#9554](https://github.com/hashicorp/terraform-provider-google/pull/9554))
+* **New Resource:** `google_network_services_edge_cache_keyset` ([#9540](https://github.com/hashicorp/terraform-provider-google/pull/9540))
+* **New Resource:** `google_network_services_edge_cache_origin` ([#9540](https://github.com/hashicorp/terraform-provider-google/pull/9540))
+* **New Resource:** `google_network_services_edge_cache_service` ([#9540](https://github.com/hashicorp/terraform-provider-google/pull/9540))
+* **New Resource:** `google_vertex_ai_featurestore_entitytype` ([#9538](https://github.com/hashicorp/terraform-provider-google/pull/9538))
+* **New Resource:** `google_vertex_ai_featurestore` ([#9538](https://github.com/hashicorp/terraform-provider-google/pull/9538))
```
Comment (c2thorn) [inline CHANGELOG.md:11]:
```suggestion
* **New Resource:** `google_vertex_ai_featurestore_entitytype` ([#9538](https://github.com/hashicorp/terraform-provider-google/pull/9538))
* **New Resource:** `google_vertex_ai_featurestore` ([#9538](https://github.com/hashicorp/terraform-provider-google/pull/9538))
```

These were beta only

**Review by c2thorn — APPROVED** [review]

**Pull request merged**